electionleaflets/settings/base.py

```diff
@@ -248,7 +248,7 @@ def setup_sentry(environment=None):
 
 DEVS_DC_AUTH_TOKEN = environ.get("DEVS_DC_AUTH_TOKEN", None)
 YNR_API_KEY = None
-YNR_BASE_URL = "https://candidates.democracyclub.org.uk"
+YNR_BASE_URL = "https://candidates.democracyclub.org.uk/api/next/"
```

Comment:
Are you sure about this?
I've not tried running this locally, but in here
https://github.com/DemocracyClub/electionleaflets/blob/f653aa4441a77db2e4af06bf56ddaea95b37f600/electionleaflets/apps/core/helpers.py#L27-L28
it looks like we're adding the "next" part of the URL in `get()`.
Should this be https://candidates.democracyclub.org.uk/api ?

This might be a place where a quick test of the `YNRAPIHelper` with [responses](https://pypi.org/project/responses/) mock might help make sure we're doing this right.

Reply:
Right, yeah, the problem here is actually `urljoin(version, endpoint)`. This joins the version string to the endpoint, but the version isn't a base URL, so the result is:

```python
>>> urljoin(version, endpoint)
'ballots'
```

Thus:

```python
>>> self.YNR_BASE
'https://candidates.democracyclub.org.uk/api/next/'
>>> urljoin(self.YNR_BASE, path)
'https://candidates.democracyclub.org.uk/api/next/ballots'
```

Whereas without changing the base URL:

```python
>>> path = f"{version}/{endpoint}"
>>> url = urljoin(self.YNR_BASE, path)
'https://candidates.democracyclub.org.uk/next/ballots'

```